### ISSUE TITLE
0.5/0.6 weirdness

### ISSUE THREAD

**Issue by quinnj** [stevengj/DecFP.jl#34]
@stevengj, any idea what's going on here? Some kind of library version error maybe?

Julia 0.5
```julia
julia> DecFP._nextfloat(one(Dec32))
+1000001E-6
```

Julia 0.6
```julia
julia> DecFP._nextfloat(one(Dec32))
+9999999E-7
```

**Comment by stevengj** [comment]
Is `one` producing the same result?

**Comment by quinnj** [comment]
On 0.6
```julia
julia> using DecFP

julia> one(Dec64)
+1E-398
```

**Comment by stevengj** [comment]
That's weird, `one` is defined as `one{T<:Number}(::Type{T}) = convert(T,1)` in both 0.6 and 0.5.

**Comment by stevengj** [comment]
I just patched DecFP for Julia 0.6; can you try it again?  It works for me.

**Comment by quinnj** [comment]
Yes, it works for me now.
